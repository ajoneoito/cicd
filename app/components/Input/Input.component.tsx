import {View, TextInput, StyleSheet} from 'react-native';
import React from 'react';

const Input = () => {
  return (
    <View>
      <TextInput placeholder="something" style={style.input} />
    </View>
  );
};

export default Input;
const style = StyleSheet.create({
  input: {
    padding: 12,
    borderWidth: 0.5,
    marginVertical: 8,
    borderRadius: 4,
    borderColor: 'grey',
  },
});
