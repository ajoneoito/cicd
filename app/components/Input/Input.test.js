import React from 'react';
import renderer from 'react-test-renderer';
import Input from './Input.component';

test('renders correctly', () => {
  const tree = renderer.create(<Input />).toJSON();
  expect(tree).toMatchSnapshot();
});