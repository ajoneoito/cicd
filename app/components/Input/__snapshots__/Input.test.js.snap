// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <TextInput
    placeholder="something"
    style={
      {
        "borderColor": "grey",
        "borderRadius": 4,
        "borderWidth": 0.5,
        "marginVertical": 8,
        "padding": 12,
      }
    }
  />
</View>
`;
